### PR TITLE
Maintenance stuff

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: tools/prepare-source.sh [hunk 1]
```diff
@@ -2,26 +2,25 @@
...
+# Curl latest composer v2 spec
+curl https://raw.githubusercontent.com/osbuild/osbuild-composer/main/internal/cloudapi/v2/openapi.v2.yml \
+     -o internal/composer/openapi.v2.yml
 
-curl https://raw.githubusercontent.com/osbuild/osbuild-composer/main/internal/cloudapi/v2/openapi.v2.yml -o internal/composer/openapi.v2.yml
-
-# go stuff
-
-GO_VERSION=1.14.14
+GO_VERSION=1.15.13
```
Comment (ondrejbudai) [inline tools/prepare-source.sh:9]: ubi has go 1.16, so we might even go with this version:

```
$ podman run --rm -it ubi8/go-toolset go version
go version go1.16.7 linux/amd64
```

Comment (Gundersanne) [reply tools/prepare-source.sh:9]: ah forgot to pull beforehand :facepalm: fixing

File: schutzbot/build.sh [hunk 2]
```diff
@@ -11,9 +11,7 @@ QUAY_REPO_URL="quay.io/osbuild/image-builder-test"
 QUAY_REPO_TAG="${CI_PIPELINE_ID}"
 
 sudo dnf install -y podman
-sudo setenforce 0 # todo
```
Comment (ondrejbudai) [inline schutzbot/build.sh:52]: such wow

**Review by ondrejbudai — APPROVED** [review]
I love this!

File: schutzbot/build.sh (hunk 2 above)
Comment (Gundersanne) [reply schutzbot/build.sh:52]: it suddenly worked without!

**Pull request merged**